README.md

```diff
@@ -46,35 +46,96 @@ Other devices may be compatible and all that is needed is to add the ID to apex-
-  - Otherwise just run `cargo build --release`
+  - Otherwise, just run `cargo build --release --features sysinfo,hotkeys,gif`
   - If you **don't** have an Apex device around at the moment or want to develop more easily you can enable the simulator: `cargo build --release --no-default-features --features crypto,clock,dbus-support,simulator`
 
 ## Configuration
 
-The default configuration is in settings.toml.
+The default configuration is in `settings.toml`.
 This repository ships with a default configuration that covers most parts and contains documentation for the important keys.
 
+You can also run the software to find errors on configuration and to decide what is the right setup you need:
+
+```shell
+$ target/release/apex-tux
+23:43:05 [INFO] Registering MPRIS2 display source.
+23:43:05 [INFO] Registering Sysinfo display source.
+23:43:05 [WARN] Couldn't find network interface `eth0`
+23:43:05 [INFO] Instead, found those interfaces:
+23:43:05 [INFO]         lo
+23:43:05 [INFO]         wlp3s0
+23:43:05 [INFO]         enp2s0
+23:43:05 [INFO]         docker0
+23:43:05 [WARN] Couldn't find sensor `hwmon0 CPU Temperature`
+23:43:05 [INFO] Instead, found those sensors:
+23:43:05 [INFO]         acpitz temp1: 67°C (max: 67°C / critical: 120°C)
+23:43:05 [INFO]         amdgpu edge: 47°C (max: 47°C)
+23:43:05 [INFO]         iwlwifi_1 temp1: 39°C (max: 39°C)
+23:43:05 [INFO]         k10temp Tctl: 66.5°C (max: 66.5°C)
+23:43:05 [INFO]         nvme Composite HFM001TD3JX013N temp1: 36.85°C (max: 36.85°C / critical: 84.85°C)
+23:43:05 [INFO]         nvme Composite Samsung SSD 980 PRO 1TB temp1: 32.85°C (max: 32.85°C / critical: 84.85°C)
+23:43:05 [INFO]         nvme Sensor 1 HFM001TD3JX013N temp2: 36.85°C (max: 36.85°C)
+23:43:05 [INFO]         nvme Sensor 1 Samsung SSD 980 PRO 1TB temp2: 32.85°C (max: 32.85°C)
+23:43:05 [INFO]         nvme Sensor 2 HFM001TD3JX013N temp3: 43.85°C (max: 43.85°C)
+23:43:05 [INFO]         nvme Sensor 2 Samsung SSD 980 PRO 1TB temp3: 38.85°C (max: 38.85°C)
+23:43:05 [INFO] Registering Clock display source.
+23:43:05 [INFO] Registering Gif display source.
+23:43:05 [INFO] Registering Coindesk display source.
+23:43:05 [INFO] Registering DBUS notification source.
+23:43:05 [INFO] Found 5 registered providers
+23:43:05 [INFO] Trying to connect to DBUS with player preference: Some("spotify")
+23:43:05 [INFO] Trying to connect to DBUS with player preference: Some("spotify")
+23:43:05 [INFO] Connected to music player: "org.mpris.MediaPlayer2.spotify"
+```
+
+In our case we need to choose a right value for the sensor(`acpitz temp1`, critical temperatured one, i.e., cpu) and the network interface(`wlp3s0`, wifi).
+
 ## Usage
 
 Simply run the binary under `target/release/apex-tux` and make sure the settings.toml is in your current directory.
 The output should look something like this:
 
-```
+```shell
 23:18:14 [INFO] register hotkey ALT+SHIFT+A
 23:18:14 [INFO] register hotkey ALT+SHIFT+D
```

Comment:
this is outdated, those messages do not appear anymore, however, maybe we should re-implement those